repos/fdbt-site/src/pages/salesConfirmation.tsx

```diff
@@ -117,10 +117,12 @@ const SalesConfirmation = ({
     </TwoThirdsLayout>
 );
 
-export const getServerSideProps = (ctx: NextPageContextWithSession): { props: SalesConfirmationProps } => {
+export const getServerSideProps = (
+    ctx: NextPageContextWithSession,
+): GetServerSidePropsResult<SalesConfirmationProps> => {
     const fareTypeAttribute = getSessionAttribute(ctx.req, FARE_TYPE_ATTRIBUTE);
-    if (!fareTypeAttribute && ctx.res) {
-        redirectTo(ctx.res, '/home');
+    if (!fareTypeAttribute) {
+        return { redirect: { destination: '/home', permanent: false } };
```

Comment:
How come this change was made?

Reply:
Basically every time we do a redirect, we also ought to be doing a return (otherwise the code continues and issues occur). In API's its easy, as we just add 'return' on the line after. However, in server side props, theres a particular return type needed to make typescript happy - which is what this is.